eoc_journal/api_client.py

```diff
@@ -0,0 +1,231 @@
+"""
+A wrapper around the Edx API.
+"""
+
+from django.conf import settings
+
+import requests
+
+
+# Points for social activities
+SOCIAL_METRICS_POINTS = {
+    'num_threads': 10,
+    'num_comments': 15,
+    'num_replies': 15,
+    'num_upvotes': 25,
+    'num_thread_followers': 5,
+    'num_comments_generated': 15,
+}
+
+PROGRESS_IGNORE_COMPONENTS = [
```

Comment:
Hmm, I really don't like having to copy all this here. We will forget to update it and get bugs/inconsistencies in the future. As part of our work to improve the progress/proficiency/engagement APIs, we should move all this logic into the platform, not Apros + XBlocks.

Question: Does Apros itself have an API that this XBlock could use to get this info, as a short term solution?